src/viem/getLockableWallet.test.ts

```diff
@@ -19,9 +19,6 @@ import {
 } from 'viem/actions'
 import { celoAlfajores, goerli as ethereumGoerli, sepolia as ethereumSepolia } from 'viem/chains'
 
-// Use real encryption
-jest.unmock('crypto-js')
```

Comment:
Real encryption is now the default, since `__mocks__/crypto-js.ts` was removed, which was mocking encrypt and decryption.